src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/UserService.kt

```diff
@@ -333,6 +346,10 @@ class UserService(
   }
 
   private fun userHasChanged(user: UserEntity, deliusUser: StaffUserDetails): Boolean {
-    return (deliusUser.email !== user.email) || (deliusUser.telephoneNumber !== user.telephoneNumber) || (deliusUser.staff.fullName != user.name) || (deliusUser.staffCode != user.deliusStaffCode) || (deliusUser.probationArea.code != user.probationRegion.deliusCode)
+    return (deliusUser.email !== user.email) ||
+      (deliusUser.telephoneNumber !== user.telephoneNumber) ||
+      (deliusUser.staff.fullName != user.name) ||
+      (deliusUser.staffCode != user.deliusStaffCode) ||
+      (deliusUser.probationArea.code != user.probationRegion.deliusCode)
```

Comment:
This won't apply if the user's team codes change (which could lead to a change in ap area). We'd have to persist these locally to determine this.

Reply:
Have agreed with Ed to persist team codes going forward (should consider backfilling these along with ap area)